Automattic/wp-calypso
# Pricing Grid: Feedback Fixes

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71546.
* This PR implements the feedback shared in p1672275676517789?thread_ts=1672225348.890109-slack-C040UAN6TPS and p1672277398019369?thread_ts=1672225348.890109-slack-C040UAN6TPS. Specifically, the following changes:
    * The Enterprise plans "... US$25,000 ..." line aligns with the billing timeframe text of the other plans.
    * The Business plan is highlighted as Popular.
* This PR also refactors all class components to functional components.

<img width="1575" alt="Screenshot 2022-12-29 at 1 00 15 PM" src="https://user-images.githubusercontent.com/1269602/209918219-9a07a317-5fb3-4782-b0f1-2d28cc3bcf0b.png">


#### Testing Instructions



* In Desktop and Tablet viewports, verify that the Enterprise plan's price aligns with the per month, billed annually text of the other plans. In Mobile, the copy should be properly padded from the previous and next rows as per Figma.
* Verify that the Business plan is highlighted as Popular in all viewports.
